features/admin.roles.v2/components/edit-role/edit-role-groups.tsx

```diff
@@ -93,6 +97,13 @@ export const RoleGroupsList: FunctionComponent<RoleGroupsPropsInterface> = (
         FederatedAuthenticatorConstants.AUTHENTICATOR_IDS.SMS_OTP_AUTHENTICATOR_ID
     ];
 
+    const enableScim2RolesV3Api: boolean = useSelector(
+        (state: AppState) => state.config.ui.enableScim2RolesV3Api
+    );
+
+    const assignGroupstoRole: (roleId: string, roleData: PatchRoleDataInterface) => Promise<any> =
```

Comment:
shall we rename as follows for consistency? please check and update the other places as well.

```suggestion
    const assignGroupstoRoleFunction: (roleId: string, roleData: PatchRoleDataInterface) => Promise<any> =
```

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/8184/commits/364f88e7a85aa677d870962f791af4ab219be515